src/resources/js/components/CreatePresentation.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <v-container>
+    <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+    <h2>Creation de presentation</h2>
+    <v-form
+      ref="form"
+      v-model="valid"
+      lazy-validation
+    >
+      <v-text-field
+        v-model="dataForm.title"
+        :counter="20"
+        :rules="titleRules"
+        label="Titre"
+        required
+      />
+
+      <v-menu
+        v-model="dateMenu"
+        :close-on-content-click="false"
+        :nudge-right="40"
+        transition="scale-transition"
+        offset-y
+        min-width="290px"
+      >
+        <template v-slot:activator="{ on }">
+          <v-text-field
+            v-model="dataForm.date"
+            label="Choisir une date"
+            :rules="dateRules"
+            required
+            readonly
+            v-on="on"
+          />
+        </template>
+        <v-date-picker v-model="dataForm.date" @input="menu = false" />
```

Comment:
Should be a datetime picker

Reply:
ok, there is no dateTime picker native in Vuetify so I planned to install [this](https://www.npmjs.com/package/vuetify-datetime-picker)

Reply:
ok ping me back when it's done

Reply:
it was done in [b3eba8](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/129/commits/b3eba88b0637e68184221285b8fc62eae1342037)